documentation/adoc/metrics.adoc

```diff
@@ -1,149 +1,137 @@
-# Metrics
+== Metrics
 
 This section describes how it's possible to deploy a Prometheus server for scraping metrics from the Kafka cluster and showing them using a Grafana dashboard. The provided resources for deploying can't be considered production grade; they are used just as an example to show how it's possible to use the metrics feature provided in this project.
 
 If you are going to use `minikube` or `minishift` for your Apache Kafka deployment adding Prometheus and Grafana servers, it would be better to increase the default amount of memory available to the virtual machine (i.e. not using the default 2 GB of RAM but 4 GB instead).
 For doing that, you can start the OpenShift cluster by running
 
-```
+[source]
 minishift start --memory 4GB
-```
 
 In case you want to use Kubernetes instead, you can run
 
-```
+[source]
 minikube start --memory 4096
-```
 
-# Deploying on OpenShift
+=== Deploying on OpenShift
 
-## Prometheus
+==== Prometheus
 
 The Prometheus server configuration uses a service discovery feature in order to discover the pods in the cluster from which it gets metrics.
 In order to have this feature working, it's necessary to provide access to the API server to get the pod list to the service account used for running the Prometheus service pod (the service account `prometheus-server` is used).
 
-```
+[source]
 export NAMESPACE=[namespace]
 oc login -u system:admin
 oc create sa prometheus-server
 oc adm policy add-cluster-role-to-user cluster-reader system:serviceaccount:${NAMESPACE}:prometheus-server
 oc login -u developer
-```
 
 where `[namespace]` is the namespace/project where the Apache Kafka cluster was deployed.
 
 After that, it's possible to deploy the Prometheus server in the cluster :
 
-1. Create the [provided "prometheus" template](prometheus/openshift-template.yaml) by running
+. Create the https://github.com/strimzi/strimzi/blob/master/metrics/prometheus/openshift-template.yaml[provided "prometheus" template] by running
++
+[source]
+oc create -f metrics/prometheus/openshift-template.yaml
 
-        oc create -f metrics/prometheus/openshift-template.yaml
+. Create a new app using the "prometheus" template:
++
+[source]
+oc new-app prometheus
 
-2. Create a new app using the "prometheus" template:
+==== Grafana
 
-        oc new-app prometheus
-
-## Grafana
-
-A Grafana server is necessary only to get a UI for the Prometheus metrics. You can use the following [project](https://github.com/OpenShiftDemos/grafana-openshift) which you need to clone locally.
+A Grafana server is necessary only to get a UI for the Prometheus metrics. You can use the following https://github.com/OpenShiftDemos/grafana-openshift[project] which you need to clone locally.
```

Comment:
Passive form again ... replace _You can use the following ..._ with something like _The following can be used ..._

Reply:
Is grafana really providing a 'UI', or would be it better to talk about visualisation of the metrics?

Reply:
Well ... hard to say ... I would be fine with saying that it provides "visualisation of the Prometheus metrics".